scripts/dts/edtlib.py

```diff
@@ -243,8 +244,7 @@ def __repr__(self):
 
 
 class Device:
-    """
-    Represents a device. There's a one-to-one correspondence between device
+    """Represents a device. There's a one-to-one correspondence between device
```

Comment:
Accidental change? The newline was deliberate.

Reply:
yeah I'll have to blame emacs `fill-paragraph` for that one